app/src/main/java/org/jellyfin/androidtv/details/DetailsActivity.kt

```diff
@@ -0,0 +1,56 @@
+package org.jellyfin.androidtv.details
+
+import android.os.Bundle
+import android.util.Log
+import androidx.fragment.app.FragmentActivity
+import androidx.leanback.app.BackgroundManager
+import com.squareup.picasso.Picasso
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.TvApp
+import org.jellyfin.androidtv.util.apiclient.asEpisode
+import org.jellyfin.androidtv.util.apiclient.getItem
+import org.jellyfin.apiclient.model.dto.BaseItemDto
+import org.jellyfin.apiclient.model.dto.BaseItemType
+import org.jellyfin.apiclient.model.dto.ImageOptions
+
+class DetailsActivity : FragmentActivity() {
+	private lateinit var fragment: BaseDetailsFragment
+
+	override fun onCreate(savedInstanceState: Bundle?) {
+		super.onCreate(savedInstanceState)
+
+		val id = intent.getStringExtra("id")
+		if (id == null) finish()
```

Comment:
Might want to add a log statement here for debugging so you won't find yourself in a 10 hour debug session asking yourself why the activity never shows up

Reply:
Ah, I think I stole that from your next episode PR, might want to add that there as well then :)

Reply:
haha good catch, I will do that